{"identifier":{"url":"doc:\/\/AsyncMoya\/documentation\/AsyncMoya\/DataError\/unknownError","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"그 외 알 수 없는 오류 상황을 처리할 때 사용합니다."}],"sections":[],"hierarchy":{"paths":[["doc:\/\/AsyncMoya\/documentation\/AsyncMoya","doc:\/\/AsyncMoya\/documentation\/AsyncMoya\/DataError"]]},"primaryContentSections":[{"kind":"declarations","declarations":[{"languages":["swift"],"platforms":["macOS"],"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"unknownError","kind":"identifier"}]}]}],"schemaVersion":{"major":0,"patch":0,"minor":3},"variants":[{"paths":["\/documentation\/asyncmoya\/dataerror\/unknownerror"],"traits":[{"interfaceLanguage":"swift"}]}],"metadata":{"externalID":"s:9AsyncMoya9DataErrorO07unknownD0yA2CmF","role":"symbol","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"unknownError"}],"roleHeading":"Case","symbolKind":"case","modules":[{"name":"AsyncMoya"}],"title":"DataError.unknownError"},"kind":"symbol","references":{"doc://AsyncMoya/documentation/AsyncMoya/DataError/unknownError":{"identifier":"doc:\/\/AsyncMoya\/documentation\/AsyncMoya\/DataError\/unknownError","title":"DataError.unknownError","role":"symbol","kind":"symbol","type":"topic","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"unknownError","kind":"identifier"}],"abstract":[{"type":"text","text":"그 외 알 수 없는 오류 상황을 처리할 때 사용합니다."}],"url":"\/documentation\/asyncmoya\/dataerror\/unknownerror"},"doc://AsyncMoya/documentation/AsyncMoya":{"kind":"symbol","title":"AsyncMoya","type":"topic","identifier":"doc:\/\/AsyncMoya\/documentation\/AsyncMoya","url":"\/documentation\/asyncmoya","abstract":[{"text":"Moya를 위한 현대적인 async\/await, Combine, RxSwift 확장 라이브러리","type":"text"}],"role":"collection"},"doc://AsyncMoya/documentation/AsyncMoya/DataError":{"title":"DataError","url":"\/documentation\/asyncmoya\/dataerror","role":"symbol","navigatorTitle":[{"kind":"identifier","text":"DataError"}],"abstract":[{"type":"text","text":"네트워크 요청 및 응답 처리 중 발생할 수 있는 다양한 오류를 표현하는 열거형"}],"kind":"symbol","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"DataError"}],"type":"topic","identifier":"doc:\/\/AsyncMoya\/documentation\/AsyncMoya\/DataError"}}}